### DISCUSSION TITLE
IP150 Firmware 4.0.4 - No site configuration

### DISCUSSION THREAD

**Discussion by ConjureOne** [ParadoxAlarmInterface/pai#390] [Q&A]
Hi, what are the required settings when IP150 is having “no site configuration”?

The IP150 is running Firmware 4.0.4. This firmware does not allow to access the command port (typically 10000) in a local network, it only allow access from SWAN using serial number.

In babyware I am using this configuration where says "IP/Panel SN (should be used if no site is created on the system)

[Screenshot](https://community.home-assistant.io/t/pai-paradox-ip150-firmware-4-0-4-no-site-configuration/615841)

In PAI I only see 2 options:

Local IP Address: firmware version < 4.0 or >= 4.40.004
SITE ID firmware versions >4.0

but no specify what happens with version = 4.0

Thank you.

**Comment by yozik04** [comment]
You want Local IP address connection. Paradox did an attempt to drop Local IP connection but then it turned out to be a big problem. Installers have started to complain about that. Paradox had to add it back. This is why there is a gap in versions.
4.0.4 > 4.0 :)

**Reply by ConjureOne** [reply]
In my version 4.0.4 local ip connection is not allowed, nor email/site. Only the one that I indicate in the link. I think this is not supported by PAI

**Reply by yozik04** [reply]
Update your IP module to the latest, you have nothing to loose. Then you will be able to use local connection.

**Comment by lpaolini** [comment]
@ConjureOne See #356.
You have the option to disable SWAN support too.
